nx-dev/ui-customers/src/lib/customer-testimonial-carousel.tsx

```diff
@@ -0,0 +1,400 @@
+                  (currentIndex - 1 + testimonials.length) % testimonials.length
+                );
+              }}
+            >
+              <ChevronLeftIcon className="h-8 w-8" />
+            </button>
+            <div
+              className="group relative h-full max-h-[450px] w-full cursor-pointer self-stretch overflow-hidden rounded-lg xl:shadow-2xl"
+              onClick={() => setIsOpen(true)}
+            >
+              {/* Thumbnail */}
+              <img
+                src={currentTestimonial.thumbnail}
+                alt={currentTestimonial.title}
+                className="aspect-video h-full w-full object-cover transition duration-300 group-hover:scale-105"
+              />
+
+              {/* Gradient Overlay */}
+              <div className="absolute inset-0 bg-gradient-to-t from-black/80 from-25% via-black/20" />
+
+              {/* Content Overlay */}
+              <div className="absolute bottom-0 left-0 p-4 md:p-8">
+                <h3 className="text-base font-semibold text-white md:text-2xl lg:text-3xl">
+                  {currentTestimonial.subtitle}
+                </h3>
+                <button className="mt-2 inline-flex items-center gap-1 rounded-full bg-white/10 px-4 py-2 text-sm font-medium text-white backdrop-blur-sm transition hover:bg-white/20 md:mt-4 md:gap-2">
+                  <PlayIcon className="size-3 md:size-5" />
+                  Watch video
+                </button>
+              </div>
+            </div>
+            {/* Next Button - Mobile only */}
+            <button
+              className="flex h-12 w-12 items-center justify-center rounded-full p-2 transition hover:text-slate-950 disabled:pointer-events-none disabled:opacity-0 md:hidden dark:hover:text-white"
+              disabled={currentIndex === testimonials.length - 1}
+              title={`Next ${testimonials[currentIndex + 1]?.company}!`}
+              onClick={() => {
+                setCurrentIndex(
+                  (currentIndex + 1 + testimonials.length) % testimonials.length
+                );
+              }}
+            >
+              <ChevronRightIcon className="h-8 w-8" />
+            </button>
+          </div>
+
+          {/* Mobile Navigation display dots */}
+          <ul className="mt-4 flex justify-center gap-2 md:hidden">
+            {testimonials.map((_, index) => (
+              <li
+                key={index}
+                className={`h-1.5 rounded-full transition-all duration-300 ${
+                  index === currentIndex
+                    ? 'w-12 bg-blue-500'
+                    : 'w-4 bg-slate-200 dark:bg-slate-700'
+                }`}
+              />
+            ))}
+          </ul>
+        </div>
+      </div>
+
+      {/* Carosel Navigation - Larger screens */}
+      <div className="relative mx-auto hidden max-w-7xl grid-cols-6 items-center justify-center px-4 pt-16 md:grid">
+        {testimonials.map(({ company, logo }, i) => (
+          <button
```

Comment:
With this implementation there is effectively no text describing what is the current logo, which impact accessibility and SEO. Please add the brand name too, can be set with the `sr-only` class.

Reply:
done